mobile/src/main/java/org/openhab/habdroid/ui/OpenHABMainActivity.java

```diff
@@ -1009,6 +910,9 @@ public void onSuccess(Call call, int statusCode, Headers headers, String respons
                     Log.d(TAG, "Command was sent successfully");
                 }
             });
+        } catch(ConnectionException e) {
+            MessageHandler.showMessageToUser(this, e.getMessage(), Constants.MESSAGES.DIALOG,
```

Comment:
Can you extend logging in debug mode like [here](https://github.com/openhab/openhab.android/pull/557/files#diff-07e4a6e281af02db331ff3f81d9f86d8R154)?

Reply:
What exactly do you mean? The second call to showmessage with DEBUG loglevel?

Reply:
See #565 for discussion.

Reply:
So, is there anything I should do here now, or should this be done in a PR, which will resolve #565 (I would prefer this, as this would implement such a functionality into all reasonable places, wouldn't it?) :)